app/helpers/ems_cluster_helper/textual_summary.rb

```diff
@@ -253,6 +253,16 @@ def textual_aggregate_disk_capacity
     {:label => "Total Disk Capacity", :value => number_to_human_size(@record.aggregate_disk_capacity.gigabytes, :precision => 2)}
   end
 
+  def textual_cinder_disk_usage
+    return nil unless @record.respond_to?(:block_storage?) && @record.block_storage?
+    {:label => "Cinder Disk Usage", :value => number_to_human_size(@record.cloud_cinder_disk_usage.bytes, :precision => 2)}
+  end
+
+  def textual_swift_disk_usage
+    return nil unless @record.respond_to?(:object_storage?) && @record.object_storage?
+    {:label => "Swift Disk Usage", :value => number_to_human_size(@record.cloud_swift_disk_usage.bytes, :precision => 2)}
+  end
```

Comment:
I still think the UI methods should be made more generic if possible for maximum pluggability.  The method names can be changed to `textual_block_storage` and `textual_object_storage` here in the UI.  I personally think the lower level methods should be renamed as well to replace "swift" and "cinder" with "object" and "block".

This way when we plug in another cloud, we don't have to create methods for every single provider's special name.

I'm not sure about the label though.  I you could use `ui_lookup` with the model class to get "Swift" or "Cinder" for the label.  @dclarizio @h-kataria Is this best way to do this?

cc @matthewd